rules/pkg-normalize.js

```diff
@@ -22,6 +22,10 @@ const isGitRepository = repository => {
 		return false;
 	}
 
+	if (typeof repository !== 'string' && !Array.isArray(repository) && Object.prototype.hasOwnProperty.call(repository, 'url')) {
```

Comment:
Hi, this works but if repository has `["SamVerschueren/clinton"]`, you condition will return true